docs/make_work.jl

```diff
@@ -170,18 +180,22 @@ function doit(
   end
 
   cd(joinpath(Oscar.oscardir, "docs")) do
-    DocMeta.setdocmeta!(Oscar, :DocTestSetup, :(using Oscar; Oscar.AbstractAlgebra.set_current_module(@__MODULE__)); recursive=true)
+    DocMeta.setdocmeta!(Oscar, :DocTestSetup, doctestsetup(); recursive=true)
```

Comment:
One more suggestion: let's modify (in a future PR, not this one) all `.md` files which currently use
````
```@setup oscar
using Oscar
```
````
to instead use
````
```@setup oscar
using Oscar
@eval Oscar.doctestsetup()
```
````
Or something like that; to ensure we get a uniform setup everywhere

Reply:
(After the modification suggested by @benlorenz )